docs/how_to_guides/logs.md

```diff
@@ -119,7 +119,7 @@ llm responses
 }
 ```
 
-For more information on `Call`, see the [History & Logs](/api_reference/history_and_logs/#guardrails.classes.history.Call) page.
+For more information on `Call`, see the [History & Logs](/docs/api_reference/history_and_logs) page.
```

Comment:
We changed the location of the API ref, can you update this to `/docs/api_reference_markdown/history_and_logs`?